premake5.lua

```diff
@@ -1051,7 +1051,7 @@ toolSharedLibrary "slang-reflection-test"
 toolSharedLibrary "render-test"
     uuid "61F7EB00-7281-4BF3-9470-7C2EA92620C3"
 
-    includedirs { ".", "external", "source", "tools/gfx", "tools/platform" }
+    includedirs { ".", "external", "external/vulkan/include", "source", "tools/gfx", "tools/platform" }
```

Comment:
Why is this needed? I am not seeing any additional changes of #include lines in this PR.

Reply:
The updated vulkan headers require this now (because they `#include <vulkan/vk_video.h>`)